contrib/util/deidentification/deidentification.php

```diff
@@ -0,0 +1,414 @@
+						"Graue","Konick","Espenshade","Machel","Noxon","Bassiti","Schnickel","Corlee","Meaker","Bolch","Iara","Laredo","Stasko","Fisette","Clar","Didlake","Borghoff","Dubberly",
+						"Quattro","Amparan","Walstrom","Mancini","Rathmanner","Andina","Muldrow","Heimbigner","Bloodough","Stoot","Bluestein","Meeks","Kaltenhauser","Ybos","Rehak","Dao","Shrum",
+						"Bjerknes","Harp","Studeny","Sweers","Granto","Eldrige","Nast","Goodling","Daquino","Allegood","Delonge","Lattrell","Willougby","Betry","Sorvig","Schremp","Waynick",
+						"Quidley","Kadner","Wares","Swarb","Placko","Travelstead","Liebold","Fukano","Daughetee","Feagler","Orie","Thruman","Quartuccio","Reinholdt","Urwin","Repoff","Crickard",
+						"Prately","Stabile","Faler","Wolfrum","Hughe","Gustave","Lemar","Zhu","Quihuiz","Krinsky","Cresswell","Vanbergen","Durelli","Carron","Targett","Emmert","Ferrusi",
+						"Harless","Ailes","Shimer","Greff","Pinna","Guedes","Meury","Lapari","Litty","Ji","Marcus","Lampel","Minotti","Migliorisi","Tevebaugh","Morgan","Steidel","Bartee",
+						"Brackman","Borruso","Ficklin","Baza","Mercier","Sponholz","Trego","Channell","Vanburen","Zaring","Luken","Komorowski","Fasciano","Drafts","Dar","Callicoat","Callam",
+						"Villaquiran","Vanbrunt","Coiner","Luckenbill","Mcray","Pin","Hayer","Lapenta","Isita","Slaydon","Frondorf","Cavagna","Passalacqua","Lehnertz","Kavadias","Macione",
+						"Sturch","Boes","Albor","Bookamer","Burbine","Bardach","Ghaor","Quartieri","Mcgill","Michelena","Aronson","Brosig","Morganti","Rodewald","Barich","Langelier"
+                  );
+
+/* function to clear the present value if a record's column and replace it with a value if spoecified
+   Input: $con, $table, $column, $value = value to replace with
+*/
+function removeColumn($con, $table, $column, $value = ''){
+    $removeSS = ("Update $table SET $column='$value' where 1 ");
+    $query = mysqli_query($con, $removeSS) or print( "\n QUERY '$removeSS' DID NOT WORK.  PLEASE VERIFY THE TABLE AND COLUMN EXISTS \n");
+    if($query) print("\n Query '$removeSS' completed! \n");
+
+}
+
+/* This function replaces the first and last name of the patient with a auto generated name
+    removes data from the encounter, removes personal information from the patient_data, notes, and form_encounter
+    tables. This should be the first function called.
+
+    INPUT: connection, last name array, firstname arrays, debug
+    OUTPUT: Number of patients DEID.
+*/
+function deIdPatientData($con, $lnames, $male, $female, $DEBUG=false){
+
+    removeColumn($con, "patient_data", "ss", "0000-00-00");
+    removeColumn($con, "form_encounter", "reason", "reason goes here, bucko");
+    removeColumn($con, "form_encounter", "facility", "Service Facility");
+    removeColumn($con, "notes", "note", "notes goes here, Captian");
+
+    $i = 0;
+
+    $removeLname = ("Select lname, pid, id, ss, street, sex from patient_data ");
```

Comment:
ss, zip+4, city, state should be removed and replaced.  ssn should simply be removed with an "if exists".  A structure query would help a lot here, or use "SELECT *" then see what you get.